packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -107,9 +145,9 @@ export const ReferenceArrayFieldView = ({
 };
 
 ReferenceArrayFieldView.propTypes = {
-    classes: PropTypes.object,
+    classes: PropTypes.any,
     className: PropTypes.string,
-    data: PropTypes.object,
+    data: PropTypes.any,
```

Comment:
why did you change this one?

Reply:
Because `RecordMap` from `ra-core` has conflict with `PropTypes.object` and I'm not sure how to resolve it yet.

Reply:
Here's the error I got from ts:
```
Type 'Requireable<object>' is not assignable to type 'Validator<RecordMap<Record>>'.
  The types of '[nominalTypeHack].type' are incompatible between these types.
    Type 'object' is not assignable to type 'RecordMap<Record>'.
      Index signature is missing in type '{}'.
```